near-sdk/src/store/lookup_set/mod.rs

```diff
@@ -1,19 +1,39 @@
 mod impls;
 
 use crate::crypto_hash::{CryptoHasher, Sha256};
-use crate::store::LookupMap;
-use crate::IntoStorageKey;
+use crate::{env, IntoStorageKey, StableMap};
 use borsh::{BorshDeserialize, BorshSerialize};
+use once_cell::unsync::OnceCell;
 use std::borrow::Borrow;
 use std::fmt;
+use std::marker::PhantomData;
+
+const ERR_ELEMENT_SERIALIZATION: &str = "Cannot serialize element";
```

Comment:
Noting there shouldn't be a need to duplicate this const. I've moved this to a common location and re-used in another PR, can always follow up later to remove this, but making comment so I don't forget.